examples/vad_transcriber/wavTranscription.md

```diff
@@ -5,7 +5,36 @@ They take in a wav file of any duration, use the WebRTC Voice Activity Detector
 to split it into smaller chunks and finally save a consolidated transcript.
 
 ### 0. Prerequisites
-Setup your environment
+#### 0.1 Install requiered packages
+Install the package which contains rec on the machine:
+
+Fedora:
+
+``` sudo dnf install sox ```
+
+Tested on: 29
+
+Ubuntu/Debian
+
+``` sudo apt install sox ```
+
+A list of distributions where the package is available can be found at: https://pkgs.org/download/sox
+
+#### 0.1 Download Deepspeech 
+
+Download a stable(!) release from the release page and extract it to a folder of your choice.
+
+This is because you need to use the same deepspeech model version and deepspeech version for things to work.
+
+You only need the example folder, but you can't download it seperately, so you have to download the whole sourcecode.
+
+For the next steps we assume you have extracted the files to ~/Deepspeech
+
+**Note: Currently there is a bug in requierement.txt of the example folders which installs deepspech 4.1 when downloading the source code for 5.1, to fix this simply run pip3 install deepspeech==0.5.1 after installing**
```

Comment:
Remove this, there is no bug. Before some 0.6 alpha, the examples were indeed dependant on 0.4.1

Reply:
I'm sorry but the last time I installed with 5.1 from the requirements it installed 4.1, so it assumed it was a bug.
Will try again and remove the note.
Nevertheless, If I try to load a 5.1 model with a cloned example folder from master it crashes while loading the model.
If I use the 5.1 release or 4.1 release it also crashes I think, but I didn't try again.
Crash happens with in the vad_transcriber_gui.py, should I open a seperate issue and remove all the warnings?

Reply:
> I'm sorry but the last time I installed with 5.1 from the requirements it installed 4.1, so it assumed it was a bug.

Yes, because the example was using 0.4.1 API.


>  Nevertheless, If I try to load a 5.1 model with a cloned example folder from master it crashes while loading the model.

Because current master requires 0.6.0a14 API now. You need a newly exported model, that's all.

Those examples are now tested on CI, so this is pretty solid.

Reply:
Ahh, I think I understand it now.
My intention was that you can simply use that step by step tutorial without having to "interact" with deepspeech, kind of the "for dummies" version.
I see now that was a bit narrow minded
Just so I'm not making the same mistake again:
When you use the examples from master, you have to "train" your own model, because I can't see a download option for the models?
When you use a alpha release there is also no model, the models are released at a specific point release?
I would rewrite to recommend to use stable release or train your own model and link to the aproppiate pages?
Or, if you prefer I can just edit it to your change requests!

Reply:
Yes and no. Yes as of now, but no because once we release 0.6 we will update the examples to stick to latest stable version, thus achieving what you want. So your doc is correct, even if too early 😊.

The rationale here is that we expect people to easily discover those examples, and that even if they don't match current master, we expect those who wants to play dirty to get dirty 😊.

Reply:
Ok, I think this commit should solve that.
I still added a note about the crashing when no model is selected and added a link to the error list.
Spelling is fixed and the bug note got removed.